code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -244,37 +251,54 @@
 	else
 		return
 
-/mob/living/simple_animal/bot/medbot/proc/tip_over(mob/user)
-	ADD_TRAIT(src, TRAIT_IMMOBILIZED, BOT_TIPPED_OVER)
-	playsound(src, 'sound/machines/warning-buzzer.ogg', 50)
-	user.visible_message(span_danger("[user] tips over [src]!"), span_danger("You tip [src] over!"))
+/*
+ * Proc used in a callback for before this medibot is tipped by the tippable component.
+ *
+ * user - the mob who is tipping us over
+ */
+/mob/living/simple_animal/bot/medbot/proc/pre_tip_over(mob/user)
+	if(!COOLDOWN_FINISHED(src, last_tipping_action_voice))
+		return
+
+	COOLDOWN_START(src, last_tipping_action_voice, 15 SECONDS) // message for tipping happens when we start interacting, message for righting comes after finishing
+	var/list/messagevoice = list("Hey, wait..." = 'sound/voice/medbot/hey_wait.ogg',"Please don't..." = 'sound/voice/medbot/please_dont.ogg',"I trusted you..." = 'sound/voice/medbot/i_trusted_you.ogg', "Nooo..." = 'sound/voice/medbot/nooo.ogg', "Oh fuck-" = 'sound/voice/medbot/oh_fuck.ogg')
```

Comment:
```suggestion
	var/static/list/messagevoice = list(
		"Hey, wait..." = 'sound/voice/medbot/hey_wait.ogg',
		"Please don't..." = 'sound/voice/medbot/please_dont.ogg',
		"I trusted you..." = 'sound/voice/medbot/i_trusted_you.ogg',
		"Nooo..." = 'sound/voice/medbot/nooo.ogg',
		"Oh fuck-" = 'sound/voice/medbot/oh_fuck.ogg',
		)
```

* `static` lists will only be initialized a single time, and then will persist. Better than rebuilding this list every time, if it will be used more than once.
* Long lists are better made in this format, it's more readable.
* Trailing commas are good, they prevent mistakes, reduce diffs and come at no cost.

Reply:
Don't static lists have a _slight_ memory cost?

Reply:
It's all trade-offs, yea.
Worrying too much about this is entering the field of micro-optimizations, though.
Either way is fineish.